tutorial_devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,52 @@
+.. _tutorial_devops_consuming_conan_center:
+
+Using ConanCenter packages in production
+========================================
+
+ConanCenter is a fantastic resource that contains a huge knowledge base generated by the community on how to create and build Conan packages for a lot of open source third party libraries and applications.
+
+It also contains a wide range of pre-compiled binaries for multiple OS, including Windows, Linux and OSX, a variety of compilers, compiler versions, static and shared libraries, etc. These precompiled packages can be very useful to do quick experiments, testing, onboarding, and they also serve as a proof that the recipes for those packages can work solidly for that wide range of configurations.
+
+However, it is not a good practice to depend directly on ConanCenter for production systems, for several reasons:
+
+- ConanCenter continuously gets contributions, fixes and improvements from the community, but that also means that ConanCenter can only guarantee that it works with the latest Conan version (see the :ref:`stability commitment in the introduction<stability>`). If you want to update your Conan version at a different pace, ConanCenter packages that introduce latest version features will break.
+- The changes in recipes might also unpexectedly cause bugs and other issues. Recipes and package updates might bump dependencies versions to use the latest, forcing a continuous update over all dependencies versions. That can cause version conflicts if your project doesn't bump dependencies versions at the same pace.
+- Fixes to ConanCenter recipes can take a while to be merged (weeks), because there are too many contributions to be reviewed. If you rely on a fix that you contributed and it is urgent for your production system, you might need to wait.
+- Binary configurations can be discontinued in ConanCenter, when new compiler versions are released and ConanCenter starts to build them, it can drop some of the older versions.
+- Outages, maintenance windows and other service interruptions can happen, ranging from a few minutes to days. The infrastructure to host central repositories is typically large and complex, and relies on cloud providers that sometimes have their own issues, networks, CDN, etc.
```

Comment:
This current PR is the section containing and explaining the mitigation, it will be just some paragraphs below. We don't have anything else in the docs at this moment, this is why I am pushing to have at least this.